pkg/runplugin/runplugin.go

```diff
@@ -202,6 +202,18 @@ func createDeploymentForConfig(config *pluginConfig) *appsv1.Deployment {
 									},
 								},
 							},
+							// NOTE This will provide WAGGLE_NODE_ID and WAGGLE_NODE_VSN for cases that a plugin
+							// needs to make a node specific choice. This is not the ideal way to manage node
+							// specific config, but may unblock things for now.
+							EnvFrom: []apiv1.EnvFromSource{
+								{
+									ConfigMapRef: &apiv1.ConfigMapEnvSource{
+										LocalObjectReference: apiv1.LocalObjectReference{
+											Name: "wes-identity",
```

Comment:
@seanshahkarami I am wondering if you tested this. The wes-identity has a suffix as you know, so I am not sure if this works with that dynamic name.

Reply:
This is likely to be wrong, but because it's in runplugin and not pluginctl I just made the comment. We should drop the runplugin stuff completely in future versions anyway.

Reply:
Oh. Sorry, I misread the filename. Yeah, that's fine. Thanks!